app/assets/javascripts/discourse/app/components/topic-list/latest-topic-list-item.gjs

```diff
@@ -0,0 +1,94 @@
+import Component from "@glimmer/component";
+import { concat, hash } from "@ember/helper";
+import { service } from "@ember/service";
+import PluginOutlet from "discourse/components/plugin-outlet";
+import PostsCountColumn from "discourse/components/topic-list/posts-count-column";
+import TopicPostBadges from "discourse/components/topic-post-badges";
+import TopicStatus from "discourse/components/topic-status";
+import UserAvatarFlair from "discourse/components/user-avatar-flair";
+import UserLink from "discourse/components/user-link";
+import avatar from "discourse/helpers/avatar";
+import categoryLink from "discourse/helpers/category-link";
+import concatClass from "discourse/helpers/concat-class";
+import discourseTags from "discourse/helpers/discourse-tags";
+import formatDate from "discourse/helpers/format-date";
+import topicFeaturedLink from "discourse/helpers/topic-featured-link";
+import topicLink from "discourse/helpers/topic-link";
+
+export default class LatestTopicListItem extends Component {
```

Comment:
Do you think this will remain separate from `<TopicListItem` in the long-run? Seems a little strange to have two similar implementations 🤔 

(historically, I think LatestTopicListItem was the pure-ember version, and TopicListItem wrapped the raw-hbs implementation?)

Reply:
They're similar, but different. 😅 From the `/styleguide`:

<img width="898" alt="image" src="https://github.com/discourse/discourse/assets/66961/ae7fbcff-4dcb-4884-a476-09bba4625688">
<img width="897" alt="image" src="https://github.com/discourse/discourse/assets/66961/8d54f999-9249-4c5d-95ab-c42f1adc3b97">

one is a table row element, the other is a div. they have different plugin outlets. one supports highlighting on navigation and bulk selection. they display slightly different sets of data.

I'd be difficult to try to merge them 😅